components/org.wso2.carbon.um.ws.service/src/main/resources/META-INF/services.xml

```diff
@@ -35,6 +35,210 @@
+        </operation>
+
+        <operation name="getUserClaimValue">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserClaimValues">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserClaimValuesForClaims">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserId">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserList">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/list
+            </parameter>
+        </operation>
+
+        <operation name="getUserListOfRole">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/rolemgt/view
+            </parameter>
+        </operation>
+
+        <operation name="isExistingRole">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/rolemgt/view
+            </parameter>
+        </operation>
+
+        <operation name="isReadOnly">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="listUsers">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/list
+            </parameter>
+        </operation>
+
+        <operation name="setUserClaimValue">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/update
+            </parameter>
+        </operation>
+
+        <operation name="setUserClaimValues">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/update
+            </parameter>
+        </operation>
+
+        <operation name="updateCredential">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/update
+            </parameter>
+        </operation>
+
+        <operation name="updateCredentialByAdmin">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/update
```

Comment:
This is an operation performed by an admin on behalf of other users as I understand. So, may be this needs a higher privilege than this.

Reply:
Agreed in mail thread